app/assets/javascripts/list_search.js.es6

```diff
@@ -89,14 +89,22 @@ App.StimulusApp.register('list-search', class extends Stimulus.Controller {
       this.selectAll()
     } else {
       this.hideAllCategories()
-      el.classList.add(ACTIVE_CLASS)
+      el.classList.add(this.ACTIVE_CLASS)
+      el.setAttribute('aria-pressed', true)
     }
     showOrHideElement(category === ALL_KEY, this.searchAllTarget)
 
     this.selectedCategories = this.activeCategories()
     this.updateCategoryContent()
   }
 
+  keyboardChangeCategory( e ) {
+    console.log(e)
```

Comment:
I think we should probably remove this console log

Reply:
Oh, definitely! I'll do that now.